mypy/semanal.py

```diff
@@ -872,6 +869,9 @@ def named_type_or_none(self, qualified_name: str) -> Instance:
             return None
         return Instance(cast(TypeInfo, sym.node), [])
 
+    def external_instance(self, qualified_name: str, args: List[Type] = None) -> Instance:
+        return Instance(self.named_type_or_none(qualified_name).type, args or None)
```

Comment:
The second argument shouldn't be `None` -- it should be an empty list if there are no arguments.